### PR TITLE
add:ci:Add sanity check script on circleci

### PR DESCRIPTION
**Pull request by aerostitch**
The idea of this is to add a sanity check before even the builds start.
This just remove the trailing spaces and runs astyle with what we did in #569 and #605.
It could be, for example, extended with the nice work @jandegr is doing on checkstyle with Java.

### CODE REVIEW

**Comment by aerostitch** [comment]
We should probably add something that enforce @jkoan 's work from https://github.com/navit-gps/navit/pull/583 in there.

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,6 +3,16 @@ defaults: &defaults
   docker:
     - image: ubuntu:14.04
 jobs:
+  sanity_check:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Setup requirements
+          command: apt-get update && apt-get -y install git astyle sed
```
Comment (jkoan) [inline .circleci/config.yml]: Please use setup_common_requirements.sh for that

Comment (aerostitch) [reply .circleci/config.yml]: Done.

Comment (aerostitch) [reply .circleci/config.yml]: Note that this step doesn't need most of what's in `setup_common_requirements.sh`. Only git, sed and astyle are needed there.

**Pull request merged**